plugins/lang-groovy/src/test/java/org/elasticsearch/messy/tests/TDigestPercentilesTests.java

```diff
@@ -383,22 +364,6 @@ public void testScriptMultiValued() throws Exception {
     }
 
     @Override
-    public void testScriptExplicitMultiValued() throws Exception {
```

Comment:
Should these tests not be left here until they are moved back too? Maybe we should just remove the @Overrides form them?